<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>简易文字编辑器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>9</y>
      <width>711</width>
      <height>491</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actFile_New"/>
    <addaction name="separator"/>
    <addaction name="actFile_Open"/>
    <addaction name="separator"/>
    <addaction name="actFile_Save"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actEdit_Cut"/>
    <addaction name="separator"/>
    <addaction name="actEdit_Copy"/>
    <addaction name="separator"/>
    <addaction name="actEdit_Paste"/>
    <addaction name="separator"/>
    <addaction name="actEdit_Clear"/>
    <addaction name="separator"/>
    <addaction name="actEdit_Undo"/>
    <addaction name="separator"/>
    <addaction name="actEdit_Redo"/>
    <addaction name="separator"/>
    <addaction name="actEdit_SelectAll"/>
    <addaction name="separator"/>
    <addaction name="actSys_ToggleText"/>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="title">
     <string>格式(&amp;M)</string>
    </property>
    <addaction name="actLang_CN"/>
    <addaction name="separator"/>
    <addaction name="actLang_EN"/>
    <addaction name="separator"/>
    <addaction name="actFont_Bold"/>
    <addaction name="separator"/>
    <addaction name="actFont_Italic"/>
    <addaction name="separator"/>
    <addaction name="actFont_UnderLine"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_M"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFile_New"/>
   <addaction name="actFile_Open"/>
   <addaction name="actFile_Save"/>
   <addaction name="separator"/>
   <addaction name="actEdit_Cut"/>
   <addaction name="actEdit_Copy"/>
   <addaction name="actEdit_Paste"/>
   <addaction name="separator"/>
   <addaction name="actEdit_Undo"/>
   <addaction name="actEdit_Redo"/>
   <addaction name="separator"/>
   <addaction name="actFont_Bold"/>
   <addaction name="actFont_Italic"/>
   <addaction name="actFont_UnderLine"/>
   <addaction name="separator"/>
   <addaction name="actLang_CN"/>
   <addaction name="actLang_EN"/>
   <addaction name="separator"/>
   <addaction name="actClose"/>
  </widget>
  <action name="actEdit_Cut">
   <property name="icon">
    <iconset theme="appointment-new" resource="images.qrc">
     <normaloff>:/prefix/images/200.bmp</normaloff>:/prefix/images/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切到粘贴板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actEdit_Copy">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/202.bmp</normaloff>:/prefix/images/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制到粘贴板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actEdit_Paste">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/204.bmp</normaloff>:/prefix/images/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>从粘贴板粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actFont_Bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/500.bmp</normaloff>:/prefix/images/500.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
  </action>
  <action name="actFont_Italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/502.bmp</normaloff>:/prefix/images/502.bmp</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="actFont_UnderLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/504.bmp</normaloff>:/prefix/images/504.bmp</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/132.bmp</normaloff>:/prefix/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭本窗口</string>
   </property>
  </action>
  <action name="actSys_ToggleText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示按钮文字</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏按钮文字</string>
   </property>
  </action>
  <action name="actEdit_Clear">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/212.bmp</normaloff>:/prefix/images/212.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
  <action name="actEdit_Undo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/206.bmp</normaloff>:/prefix/images/206.bmp</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销上次编辑操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actEdit_Redo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/208.bmp</normaloff>:/prefix/images/208.bmp</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做上次操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actEdit_SelectAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actFile_New">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/100.bmp</normaloff>:/prefix/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actFile_Open">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/122.bmp</normaloff>:/prefix/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actFile_Save">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/104.bmp</normaloff>:/prefix/images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actLang_CN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/CN.jpg</normaloff>:/prefix/images/CN.jpg</iconset>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
   <property name="toolTip">
    <string>中文界面</string>
   </property>
  </action>
  <action name="actLang_EN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/prefix/images/timg2.jpg</normaloff>:/prefix/images/timg2.jpg</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
   <property name="toolTip">
    <string>English Interface</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
